services/github.js

```diff
@@ -0,0 +1,78 @@
+const {GraphQLClient} = require('graphql-request');
+// graphgl-request 3.1.0 workaround
+// https://github.com/prisma-labs/graphql-request/issues/206
+const { Headers } = require('cross-fetch');
+global.Headers = global.Headers || Headers;
+// End of workaround
+
+const config = require('../config');
+
+exports.searchRepositories = async (queryString) => {
+
+  const endpoint = 'https://api.github.com/graphql';
+  
+  const graphQLClient = new GraphQLClient(endpoint, {
```

Comment:
IMHO the client initialization can be moved to the file scope as well, so we can reuse the same instance for both of the exported functions (`searchRepositories`, `getContributors`) since the `config. githubToken` won't change and we should not support multiple tokens at this point.